settings.json

```diff
@@ -48,166 +48,168 @@
                 "progresses": {},
                 "next_update": 0
             }
-        }
+        },
+        "halloween_treats": 0
```

Comment:
I understand that. but it's temp event for like 10 days, thought of just unsetting/deleting that field once the event ends. Don't actually want to change anything in user's table. Thought of making a new collection, but since that would be too much for such a small event, I've made it this way. I can change if we wanna do it in game state, but this state wouldn't matter after event end. so that would be a stale data.